README.md

```diff
@@ -19,6 +19,7 @@ See [tags/releases](https://github.com/smart-on-fhir/Swift-FHIR/releases).
 
  Version |   Swift   |      FHIR     | &nbsp;
 ---------|-----------|---------------|-----------------------------
+**4.0**|       4.0 | `3.3.0.13671` | STU 4
```

Comment:
Since this is still a ballot version of R4, can you make this version `3.3` so we can use version number `4.0` for the official R4 release?